api/src/services/versions.ts

```diff
@@ -253,6 +258,18 @@ export class VersionsService extends ItemsService {
 			delta: revisionDelta,
 		});
 
+		let existingDelta = version['delta'];
+
+		if (!existingDelta) {
+			const saves = await this.getVersionSavesById(key);
+
+			existingDelta = assign({}, ...saves);
+		}
+
+		const finalVersionDelta = assign({}, existingDelta, revisionDelta ? JSON.parse(revisionDelta) : null);
+
+		await this.updateOne(key, { delta: finalVersionDelta });
```

Comment:
I'm leaning towards the breaking change by requiring admins to update permissions. 🤔

Reply:
In general, it sounds like a good idea - especially as it would allow to separately define permissions for _save_ (until now only controlled via item & version _read_ permissions) - but, that way, we could not differentiate between _save_ and version _update_ permissions.

In general this sounds like a good idea - especially because it would allow to define the rights for _save_ separately (so far only controlled via item & version _read_ permissions) - but this way we could not distinguish between permissions for _save_ and _update_ of versions.

For this reason, I think we should simply disregard the permissions for the update call. As we only transmit the delta, I see no problem with this.
For now, I have adapted it accordingly in c9573417f22c0e8307f3eee8248f4dd219008d4b. Please have a look at it 🙏